### ISSUE TITLE
[BUG] Cannot launch electron app from command line

### ISSUE THREAD

**Issue by Strikingwolf** [atom/electron#2158]
I'm currently trying to experiment with electron by making an irc client. When I try to run in the command line it fails on require app with the message https://paste.ee/p/rkEUp. I don't understand why it wouldn't be able to find the module because i have electron installed both globally and as a node-module in the directory, any ideas?

**Comment by maxogden** [comment]
@Strikingwolf can you supply the info requested here? https://github.com/atom/electron/blob/master/CONTRIBUTING.md#submitting-issues

specifically these:

- The version of Electron you are using
- The operating system you are using
- What command you ran to get the error

**Comment by zcbenz** [comment]
Can you add `console.log(module.paths, process.env.NODE_PATH)` to the head of your main script and paste the result? And does the downloaded binary from https://github.com/atom/electron/releases work?

This is very likely caused by some environment settings on your machine.

**Comment by Strikingwolf** [comment]
I'll be able to supply he info requested to yall tomorrow, not on my computer currently

**Comment by Strikingwolf** [comment]
Electron version: 0.4.1
OS X
electron ./ and also /Applications/Electron.app/Contents/MacOS/Electron ./ in the applications directory

console.log(module.paths, process.env.NODE_PATH) returns https://paste.ee/p/scJZH

The binary does work

And now that I ran that console log I figured out the error. I was running the command using term2 for Atom. If I run in a regular terminal it works. From now on I'll run that console.log on all my electron stuff to make sure an editor isn't fudging the node stuff. Thanks for the help!
